mythril/laser/ethereum/keccak_function_manager.py

```diff
@@ -33,6 +33,7 @@ def __init__(self):
         self.store_function = {}  # type: Dict[int, Tuple[Function, Function]]
         self.interval_hook_for_size = {}  # type: Dict[int, int]
         self._index_counter = TOTAL_PARTS - 34534
+        self.store_hashes = {}  # type: Dict[int, List[BitVec]]
```

Comment:
What about `hash_result_store`. (this is fairly subjective so I don't mind if you don't do anything with this comment)
This seems slightly more clear on the purpose of the variable

Reply:
I am in class now. Changing this on mobile phone is gonna be tricky, feel free to commit it to this pr( or I can do it few hrs later if there is no urgency)